orchagent/portsorch.cpp

```diff
@@ -1380,7 +1434,10 @@ bool PortsOrch::setPortAdminStatus(Port &port, bool state)
         SWSS_LOG_ERROR("Failed to set admin status %s for port %s."
                        " Setting host_tx_ready as false",
                        state ? "UP" : "DOWN", port.m_alias.c_str());
-        m_portStateTable.hset(port.m_alias, "host_tx_ready", "false");
+        if (!m_cmisModuleAsicSyncSupported)
```

Comment:
do you still need this if?

Reply:
why would I not need it? 
the purpose of it is to set host_tx_ready=false by default only when we work in dependent mode.